accounting/fiscal_localizations/overview/localizations_list.rst

```diff
@@ -0,0 +1,126 @@
+===========================
+List of supported countries
+===========================
+
+Odoo Accounting can be used in many countries out of the box by installing the appropriate module.
+Here is a list of all :doc:`Fiscal Localization Packages <fiscal_localization_packages>` that are
+available on Odoo.
+
+.. image:: media/fiscal_localization_packages_modules.png
+   :align: center
+   :alt: Odoo Accounting.
+
+- LATAM Localization Base
+- U.A.E. - Accounting
+- Argentina
+
+  - Argentinian Accounting
+  - Argentinian Accounting Reports
+  - Argentinian Electronic Invoicing
+
+- Austria - Accounting
+- Australian - Accounting
+- Belgium - Accounting
+- Bolivia - Accounting
+- Brazilian - Accounting
+- Canada - Accounting
+- Switzerland - Accounting
+- Chile - Accounting
+- China
+
+  - China- Accounting
+  - China - City Data
+  - China - Small Business CoA
+  - China - Standard CoA
+
+- Colombia
+
+  - Colombian - Accounting
+  - Colombian - Point of Sale
+
+- Costa Rica - Accounting
+- Germany
+
+  - Germany - Accounting
+  - Germany SKR03 - Accounting
+  - Germany SKR04 - Accounting
+
+- Denmark
+
+  - Denmark - Accounting
+  - Denmark - Accounting Reports
+
+- Dominican Republic - Accounting
+- Algeria - Accounting
+- Ecuador - Accounting
+- Spain - Accounting (PGCE 2008)
+- Ethiopia - Accounting
+- EU Mini One Stop Shop (MOSS)
+- Finnish Localization
+- France - Accounting
+- Generic - Accounting
+- Greece - Accounting
+- Guatemala - Accounting
+- Hong Kong - Accounting
+- Honduras - Accounting
+- Croatia - Accounting (RRIF 2012)
+- Hungarian - Accounting
+- Indonesian - Accounting
+- Ireland - Accounting
+- Israel - Accounting
+- India
+
+  - Indian - Accounting
+  - Indian - Point of Sale
+
+- Italy
+
+  - Italy - Accounting
+  - Italy - E-invoicing
+
+- Japan - Accounting
+- LATAM Document
+- Lithuania
+
+  - Lithuania - Accounting
+  - LT - Accounting Reports
+
+- Luxembourg - Accounting
+- Maroc - Accounting
+- Mongolia
+
+  - Mongolia - Accounting
+  - Mongolia - Accounting Reports
+
+- Multi Language Chart of Accounts
+- Mexico - Accounting
+- Netherlands
+
+  - Netherlands - Accounting
+  - Netherlands Intrastat Declaration
+
+- Norway - Accounting
+- New Zealand - Accounting
+- Panama - Accounting
+- Peru - Accounting
+- Poland - Accounting
+- Portugal - Accounting
+- Romania - Accounting
+- Saudi Arabia - Accounting
+- Swedish - Accounting
+- Sweden - Structured Communication OCR
+- Singapore - Accounting
+- Slovenian - Accounting
+- OHADA - Accounting
+- Thailand - Accounting
+- Turkey - Accounting
+- Ukraine - Accounting
+- UK - Accounting
+- United States - Accounting
+- Uruguay - Chart of Accounts
+- Venezuela - Accounting
+- Vietnam - Accounting
+- South Africa - Accounting
+
```

Comment:
Alphabetical order? Easier to find the information nope? :)

Reply:
.... of course ^^ you're right.